packages/fxa-settings/src/styles/tailwind.scss

```diff
@@ -38,3 +37,27 @@ body {
     @apply transition duration-150;
   }
 }
+
```

Comment:
I tried avoiding any non tailwind classes, but the only way I could find to use `:before` or other pseudo elements was to [use it in a plugin](https://github.com/yutahaga/tailwindcss-pseudo-elements), I tried it out but it seemed to be a much worse solution as you have to add each rule as a `utility`. https://github.com/yutahaga/tailwindcss-pseudo-elements#configuration

Reply:
I also noticed that some tooltips anchor from the center of the element:

<img width="149" alt="Screen Shot 2020-09-18 at 11 59 15 AM" src="https://user-images.githubusercontent.com/6392049/93621619-d49bf000-f9a9-11ea-8506-c3adb46e24de.png">

While others anchor from the left:

<img width="253" alt="Screen Shot 2020-09-18 at 11 59 05 AM" src="https://user-images.githubusercontent.com/6392049/93621635-dc5b9480-f9a9-11ea-97db-cd97e58695a7.png">

I could see this being a prop on the component that uses different anchoring classes.